.travis.yml

```diff
@@ -0,0 +1,49 @@
+# You must sign into travis-ci.org and set the commit hook on your project for travis to
+# run on your project. The secure: variable must be generated by running `travis encrypt`
+# on a github oauth key that you can generate using curl.
+
+sudo: true
+language: node_js
+env:
+  global:
+    - FF_BETA_PATH: './firefox/firefox'
+cache:
+  directories:
+    - node_modules
+node_js:
+  - 'stable'
+addons:
+  # You can't install latest and beta as addons
+  firefox: latest
+  # Current broken on Travis 15/March/2016
+  #apt:
+  #  sources:
+  #    - google-chrome
+  #  packages:
+  #    - google-chrome-stable
+  #    - google-chrome-beta
+
+before_install:
+  # Chrome Stable
+  - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update -qq
+  - sudo apt-get install -y google-chrome-stable
+
+  # Beta currently has dependencies that can't be met
+  # - sudo apt-get install -y google-chrome-beta
+
+  # Get firefox beta and extract to tar for the test runner to use.
+  - wget 'https://download.mozilla.org/?product=firefox-beta-latest&lang=en-US&os=linux64' -O firefox-beta.tar.bz2 && tar xvjf firefox-beta.tar.bz2
+
+install:
+  - npm install
+  - npm install -g esdoc
```

Comment:
How's the global `esdoc` being used? Can you switch it to a local `esdoc` that's invoked via a `npm run`? That way you'd take advantage of the caching of `node_modules`.